cmd/bridge/main.go

```diff
@@ -295,6 +295,11 @@ func main() {
 		apiServerEndpoint = srv.K8sProxyConfig.Endpoint.String()
 	}
 	srv.KubeAPIServerURL = apiServerEndpoint
+	srv.K8sClient = &http.Client{
```

Comment:
What does this stanza do in relation to deleting the token?

Reply:
It creates an HTTP client that is used to talk to the API server

Reply:
Right, I get it's creating a client. Surprised we didn't already have one to reuse.  I need to work through the rest of the file a bit more.